docs/tensorflow.md

```diff
@@ -15,7 +15,7 @@
 ### Versions
 - Zero Script Change experience where you need no modifications to your training script is supported in the official [SageMaker Framework Container for TensorFlow 1.15](https://docs.aws.amazon.com/sagemaker/latest/dg/pre-built-containers-frameworks-deep-learning.html), or the [AWS Deep Learning Container for TensorFlow 1.15](https://aws.amazon.com/machine-learning/containers/).
 
-- This library itself supports the following versions when you use our API which requires a few minimal changes to your training script: TensorFlow 1.13, 1.14, 1.15. Keras 2.3.
+- This library itself supports the following versions when you use our API which requires a few minimal changes to your training script: TensorFlow 1.14, 1.15, 2.0.1, 2.1.0. Keras 2.3.
```

Comment:
why 1.13 support was removed? 
Is it not supported and things break now for 1.13?

Reply:
1.13 is broken with smdebug 0.6.0 as well (before this PR). there was a discussion about whether to add support for 1.13 or discontinue, and the decision was made to discontinue. maybe @ddavydenko could throw more light on the discussion.

Reply:
We discussed with @satadalb and agreed that it doesn't make sense to fix support for 1.13 given that we are now introducing support for 2.x and more and more customers will migrate toward TF 2.x.

Reply:
Can you give me an idea about what was broken in 1.13 ?

Reply:
Have we assessed the potential regression impact to sagemaker customers? 
If there is sagemaker training job with tf 1.13, it will stop working as soon as this version gets picked up.

Reply:
Pasting offline chat. Given that DLC TF 1.13 didn't have ZCC, there is no regression here as no customer would be using DLC TF 1.13

[March 9, 2020, 2:09 PM] Davydenko, Denis: Q: - 1.13 was never running in ZCC mode- Didn’t we produce DLC with TF 1.13 + smdebug ? 
A: Nope, only 1.15 container has ZCC
[March 9, 2020, 2:10 PM] Davydenko, Denis: Q: - PR 150 has smell of potential regression for SM customers. What are the version of TF in ZCC + TF DLC containers that we have today out in public? 
A: that would be correct in general. However, today no 1.13 customers could use smdebug. So we are not introducing regression with 0.7.0. today out in public we have 1.15 TF DLC container with ZCC changes for smdebug in it